### PR TITLE
Updated rules.rst and actionchain.rst for typos

### PR DESCRIPTION
**Pull request by humblearner**

### CODE REVIEW

File: docs/source/actionchain.rst [hunk 1]
```diff
@@ -21,11 +21,11 @@ Following is sample ActionChain workflow definition named :github_st2:`echochain
 Details:
 
 * ``chain`` is the array property that contains tasks, which incapsulate action invocation.
-* Tasks are named action execution specifications. The name is scoped to an ActionChain and is used as a reference to a task.
+* Tasks are named action execution specifications, seperated by ``-``. The name is scoped to an ActionChain and is used as a reference to a task.
```
Comment (LindsayHill) [inline docs/source/actionchain.rst]: s/seperated/separated/

Comment (manasdk) [inline docs/source/actionchain.rst]: by ``-`` do you mean the list representation in yaml?

File: docs/source/rules.rst [hunk 2]
```diff
@@ -456,37 +456,37 @@ did not match or one of the criteria.
 If you are debugging and would like to see the list of trigger instances sent to |st2|,
 you can use the CLI to do so:
 
-..  code-block:: bash
+.. code-block:: bash
 
-  st2 triggerinstance list
+  st2 trigger-instance list
```
Comment (manasdk) [inline docs/source/rules.rst]: !

**Comment by manasdk** [comment]
👍

File: docs/source/actionchain.rst (hunk 1 above)
Comment (humblearner) [inline docs/source/actionchain.rst]: yes!!

Comment (manasdk) [inline docs/source/actionchain.rst]: Ah! Something like 

"Tasks are named action execution specifications provided in a list."

**Comment by manasdk** [comment]
Greatly appreciate your efforts to get us out of linguistic squalor :)

**Comment by humblearner** [comment]
This is what you do on side when firmware download feature blocks you for couple of hours: Read and fix along the way!!

**Pull request merged**